script.js

```diff
@@ -210,15 +209,22 @@ var Phonebook = {
 
 var MutateMeNot = [1, 2, 3, 4, 5];
 
+var userFirstNumber = prompt("Enter your first number between 0-4");
+var userSecondNumber = prompt("Enter your second number between 0-4");
 
+userFirstNumber = parseInt(userFirstNumber);
+userSecondNumber = parseInt(userSecondNumber);
 
+var newArray = MutateMeNot;
```

Comment:
A small comment like this will not be sufficient, but this is the point of this extra exercise. newArray here stores only a reference to MutateMeNot. After what you've done below, try to console.log the value of MutateMeNot and you will see that it has also changed because everything you've done to newArray is also being done to MutateMeNot since newArray refers to MutateMeNot.